src/CallableResolver.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Middlewares\Utils;
+
+use RuntimeException;
+
+/**
+ * Simple interface to resolve callableish values.
+ */
+interface CallableResolver
```

Comment:
I know that many people don't like it but in Psr, all interfaces contains the suffix `Interface`, so I'd like to be compliant with this stantard and use the name `CallableResolverInterface`.